packages/next-app/src/components/Input.tsx

```diff
@@ -19,9 +19,11 @@ export const Input = React.forwardRef<HTMLInputElement, IInput>(
             "w-full selection:color-white box-border inline-flex h-[35px] appearance-none items-center justify-center rounded-[4px] bg-blue4 px-[10px] text-[15px] leading-none text-slate12 shadow-[0_0_0_1px] shadow-blue6 outline-none selection:bg-blue9 hover:shadow-[0_0_0_1px_black] focus:shadow-[0_0_0_2px_black] disabled:bg-blue1"
           )}
         />
-        <div className="h-6 mt-1 text-red-800">
-          {errorMessage && <span>{errorMessage}</span>}
-        </div>
+        {errorMessage && (
+          <div className="h-6 mt-1 text-red-800 text-xs">
```

Comment:
I know this was not added by your changes, but prefer to using [radix colors](https://www.radix-ui.com/colors) here as that should be the convention we've adopted elsewhere in the app